WordPress/Classes/ViewRelated/Themes/ThemeBrowserCell.swift

```diff
@@ -0,0 +1,125 @@
+import Foundation
+
+public class ThemeBrowserCell : UICollectionViewCell {
+    
+    // MARK: - Outlets
+    
+    @IBOutlet weak var imageView : UIImageView!
+    @IBOutlet weak var nameLabel : UILabel!
+    @IBOutlet weak var infoLabel: UILabel!
+    @IBOutlet weak var highlightView: UIView!
+    @IBOutlet weak var activityView: UIActivityIndicatorView!
+    
+    // MARK: - Properties
+    
+    public var theme : Theme? {
+        didSet {
+            refreshGUI()
+        }
+    }
+    
+    private var placeholderImage = UIImage(named: "theme-loading")
+    
+    // MARK: - GUI
+    
+    /**
+    @brief Calculates cell height fitting a given width
+    
+    @param width    Intended width
+    @return         Matching height
+    */
+    public class func heightForWidth(width: CGFloat) -> CGFloat {
+        
+        let imageHeight = (width - WPStyleGuide.Themes.cellImageInset) * WPStyleGuide.Themes.cellImageRatio
+        let height = imageHeight + WPStyleGuide.Themes.cellInfoBarHeight
```

Comment:
**Tip, no need to change anything!**

```
private typealias Styles = WPStyleGuide.Themes
```

(And that would enable you to do something like: `Styles.cellImageInset / Styles.cellInfoBarHeight`)